modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2936,6 +2936,70 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the Formula Builder is not searchable"
+	@priority = 3
+	test FormulaFieldIsNotSearchable {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given: A Custom Object with two fields") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 176845",
+				objectName = "CustomObject176845",
+				pluralLabelName = "Custom Objects 176845");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176845");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Integer 1",
+				fieldType = "Integer");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Integer",
+				fieldLabelName = "Custom Integer 1",
+				fieldName = "customInteger1",
+				fieldType = "Integer",
+				isRequired = "false",
+				objectName = "CustomObject176845");
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Formula Field",
+				fieldType = "Formula",
+				formulaFieldOption = "Integer");
+
+			ObjectAdmin.goToFieldsDetails(label = "Custom Formula Field");
+
+			Type(
+				key_labelName = "Formula Builder",
+				locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT",
+				value1 = "customInteger1 + customInteger1");
+
+			Button.clickSave();
```

Comment:
Since we have multiple tests setting up the custom formula field, we should create a macro to setup the custom formula field. Keep in mind that we don't want to navigate to the field details in the macro, we only want to edit it.

@davisaints please ask in our channel if you need help or more details. We can provide examples for you so you can see what I mean.

Reply:
Well watched. I'll make those changes, thanks.